tensorflow/python/data/kernel_tests/test_base.py

```diff
@@ -132,7 +137,7 @@ def _compareOutputToExpected(self, result_values, expected_values,
     for i in range(len(result_values)):
       nest.assert_same_structure(result_values[i], expected_values[i])
       for result_value, expected_value in zip(
-          nest.flatten(result_values[i]), nest.flatten(expected_values[i])):
+              nest.flatten(result_values[i]), nest.flatten(expected_values[i])):
```

Comment:
undo formatting change to avoid lint errors

Reply:
addressed in the commit